src/interface/swagger.json

```diff
@@ -634,6 +634,12 @@
           ],
           "description": "PISP identifier who uses this Consent"
         },
+        "status": {
```

Comment:
swagger.json should be generated using swagger-cli from api-template

Reply:
not completely sure how to do that